src/compas_ags/rhino/forceobject.py

```diff
@@ -121,7 +122,18 @@ def draw(self):
                         color[edge] = self.settings['color.compression']
 
             guids = self.artist.draw_edges(color=color)
-            self.guid_edge = zip(guids, edges)
+            if len(guids) == len(edges):
+                self.guid_edge = zip(guids, edges)
+            else:
+                drawn_edges = []
+                for edge in edges:
+                    u, v = edge
+                    pt1, pt2 = self.diagram.edge_coordinates(u, v)
+                    if geometric_key(pt1) == geometric_key(pt2):
```

Comment:
this fix the bug @Licini . I only store in the guide:edge dictionary the drawn edges guide:drawn_edges

Reply:
This can also happens in some instances of the form diagram. so I repeated the same code for `formobject.py`

Reply:
cool looks good!

Reply:
this is a fragile solution though because it assumes that edges are not drawn when the geometric keys of start and end are the same, but that is not the criterion for not drawing edges...

Reply:
Once we receive `None` from compas_rhino I can update this.